src/pytsql/grammar/adjust_antlrs_grammar.py

```diff
@@ -0,0 +1,76 @@
+import click
```

Comment:
I find it interesting that it seems pre-commit is not doing its job because this is a violation of isort config.